### ISSUE TITLE
Button not compatible with screen readers 

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1623]
In the 'settings/account' menu, under 'account deletion,' the 'show password' feature is incorrectly implemented as a `span` element rather than a button and lacks a readable label. Additionally, the requirement for users to keep pressing the `span` to activate the feature is not compatible with how screen readers interact with content, rendering the functionality inaccessible to users who rely on screen readers.
